postgres-openssl/src/lib.rs

```diff
@@ -134,13 +134,15 @@ where
     S: AsyncRead + AsyncWrite + Unpin + Debug + 'static + Sync + Send,
```

Comment:
You can remove the `Debug + 'static + Sync` bounds from here now.

Reply:
Ok, `Debug` and `Sync` are gone. Compiler still insists on `'static` though.